### PR TITLE
Add minimal styleguide to showcase

### PR DESCRIPTION
**Pull request by meodai**
Add a minimal style-guide to the showcase that explains and shows the commonly used visual patterns. (Does not include components that are only used once)

### CODE REVIEW

File: src/frontend/src/styleguide.ts [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+          can be used to style lead paragraphs that follow a title. The
+          <code>.t-paragraph</code> class is a more generic class that can be
+          used to style any other paragraph.
+        </p>
+      </aside>
+      <aside class="l-section demo-section">
+        <h2 class="t-title t-title--sub">Various text helpers</h2>
+        <section class="demo" aria-label="Verious elements Demo">
+          <p class="t-paragraph">
+            <a href="#">This is an actual a tag</a><br />
```
Comment (nmattia) [inline src/frontend/src/styleguide.ts:107]:
```suggestion
            <a href="#">This is an actual "a" tag</a><br />
```

File: src/frontend/src/styleguide.ts [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+            ><br />
+            <a href="#" class="t-link"
+              ><i class="t-link__icon">+</i>Link with icon</a
+            ><br />
+          </p>
+        </section>
+        <p class="t-lead">
+          There are a bunch of other text helpers that can be used to style text
+          within a paragraph or anywhere else. One of the rare elements that has
+          a default style is a <a href="#">link</a> But sometimes you might want
```
Comment (nmattia) [inline src/frontend/src/styleguide.ts:122]:
```suggestion
          a default style is a <a href="#">link</a>, but sometimes you might want
```

File: src/frontend/src/styleguide.ts [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+          an other element to look like a link. In that case you can use the
+          <code>.t-link</code> class. In some rare cases we don't want links to
```
Comment (nmattia) [inline src/frontend/src/styleguide.ts:124]:
```suggestion
          <code>.t-link</code> class. In some rare cases we want links to
```

File: src/frontend/src/styleguide.ts [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+          look less prominent (by removing the underline) you can use the
+          <code>.t-link--discreet</code> class.
+        </p>
+      </aside>
+    </article>
+
+    <article class="l-section  c-card c-card--highlight">
+      <h1 class="t-title t-title--main">Layout</h1>
+      <p class="t-lead">
+        We use a bunch of layout classes to style the .c-card__icon of the app.
```
Comment (nmattia) [inline src/frontend/src/styleguide.ts:134]:
```suggestion
        We use a bunch of layout classes to style the <code>.c-card__icon</code> of the app.
```

File: src/frontend/src/styleguide.ts [hunk 5]
```diff
@@ -0,0 +1,282 @@
+/** A shocase of common CSS patterns that can be reuses all all over the app */
```
Comment (mstrasinskis) [inline src/frontend/src/styleguide.ts:1]:
```suggestion
/** A showcase of common CSS patterns that can be reuses all all over the app */
```

File: src/frontend/src/styleguide.ts [hunk 6]
```diff
@@ -0,0 +1,282 @@
...
+      <p class="t-lead">
+        We use a bunch of layout classes to style the .c-card__icon of the app.
+        They just define where elements are placed.
+      </p>
+
+      <aside class="l-section demo-section">
+        <h2 class="t-title t-title--sub">Sections</h2>
+        <p class="t-lead">
+          A section, much like a text-paragraph, is a visually distinct area of
+          the app. In our case in just sets a space to the top. It can be used
```
Comment (nmattia) [inline src/frontend/src/styleguide.ts:142]:
```suggestion
          the app. In our case it just sets a space to the top. It can be used
```

**Review by nmattia — COMMENTED** [review]
Nice, thanks! Few comments

**Review by nmattia — APPROVED** [review]
Looks great, thanks for doing this!

**Pull request merged**